pkg/storkctl/clusterpair.go

```diff
@@ -490,30 +427,20 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 				destToken = token
 			}
 
-			srcClusterPair, err := generateClusterPair(clusterPairName, cmdFactory.GetNamespace(), dIP, dPort, destToken, dFile, projectMappingsStr, pxAuthSecretNamespaceDest, false, false)
+			sconf, err := getConfig(sFile).ClientConfig()
 			if err != nil {
 				util.CheckErr(err)
 				return
 			}
-
-			// Create cluster-pair on source cluster
-			conf, err := getConfig(sFile).ClientConfig()
+			dconf, err := getConfig(dFile).ClientConfig()
 			if err != nil {
 				util.CheckErr(err)
 				return
 			}
 
-			if len(cmdFactory.GetNamespace()) > 0 {
-				err = createNamespace(sFile, cmdFactory.GetNamespace())
-				if err != nil {
-					util.CheckErr(err)
-					return
-				}
-			}
-			printMsg(fmt.Sprintf("\nCreating Secret and ObjectstoreLocation in source cluster in namespace %v...", cmdFactory.GetNamespace()), ioStreams.Out)
-			storkops.Instance().SetConfig(conf)
-			core.Instance().SetConfig(conf)
-			secret := &v1.Secret{
+			finalBackupLocationName := backupLocationName
```

Comment:
The finalBackupLocationName is not used on line 446 when creating the BackupLocation CR?

Reply:
finalBackupLocationName will be removed. The reason for not using that in case of creating backupLocation CR was because CR creation is only required when the user input `backuplocation` is not provided. In that case it is same as clusterpair name.